ydb/core/cms/cms_tx_load_state.cpp

```diff
@@ -9,6 +9,21 @@
 
 namespace NKikimr::NCms {
 
+using namespace NKikimrCms;
+
+namespace {
+
+template<typename T>
+T ParseFromString(const TString& str) {
+    google::protobuf::TextFormat::Parser parser;
+    parser.AllowUnknownField(true);
+    T result;
+    parser.ParseFromString(str, &result);
+    return result;
```

Comment:
Я бы сохранил семантику `ParseFromString`, принимая `T` по ссылке/указателю:
```suggestion
bool ParseFromStringSafe(const TString& input, T* output) {
    google::protobuf::TextFormat::Parser parser;
    parser.AllowUnknownField(true);
    return parser.ParseFromString(str, output);
```